openapi/data/exc.py

```diff
@@ -5,16 +5,25 @@
 
 
 @dataclass
-class FieldError:
+class ErrorMessage:
+    """Error message and list of errors for data fields
+    """
+    message: str = data_field(description='Error message')
+
+
+@dataclass
+class FieldError(ErrorMessage):
     """Error message for a data field
     """
     field: str = data_field(description='name of the data field with error')
-    message: str = data_field(description='error message')
 
 
 @dataclass
-class Error:
+class Error(ErrorMessage):
```

Comment:
Can we rename to `ErrorList` or `ValidationErrors` or something more descriptive?